File: tests/e2e/fixtures/bookstore/mysql/java/src/main/java/com/example/db/com/example/db/models/Product.java
package com.example.db.models;

public record Product(
    String id,
    String sku,
    String name,
    boolean active,
    Float weightKg,
    Double rating,
    String metadata,
    byte[] thumbnail,
    java.time.LocalDateTime createdAt,
    short stockCount
) {}

File: tests/e2e/fixtures/bookstore/mysql/kotlin/src/main/kotlin/com/example/db/com/example/db/models/Product.kt
package com.example.db.models

data class Product(
    val id: String,
    val sku: String,
    val name: String,
    val active: Boolean,
    val weightKg: Float?,
    val rating: Double?,
    val metadata: String?,
    val thumbnail: ByteArray?,
    val createdAt: java.time.LocalDateTime,
    val stockCount: Short
)
